source/layouts/layout.html.haml

```diff
@@ -2,7 +2,7 @@
   <!--[if IE]><script src='http://html5shiv.googlecode.com/svn/trunk/html5.js'></script><![endif]-->
   %meta{charset:'utf-8'}
   <meta http-equiv='X-UA-Compatible' content='IE=edge,chrome=1'>
-  %meta{content:'width=device-width,initial-scale:1.0,maximum-scale:1.0', name:'viewport'}
+  %meta{content:'width=device-width', name:'viewport'}
```

Comment:
`initial-scale=1,maximum-scale=1` would probably work. It should be an equals sign, not a colon.